src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck is working right now!");
```

Comment:
The message in the `doWork` method should be more informative and indicate the specific type of machine that is working. Consider including the class name in the message.